app/src/main/java/com/jerboa/ui/components/common/TimeAgo.kt

```diff
@@ -40,6 +42,15 @@ fun TimeAgo(
 ) {
     val publishedPretty = dateStringToPretty(published, longTimeFormat)
 
+    if (publishedPretty == null) {
+        Text(
```

Comment:
Since this is used twice, you might as well extract it into a composable.

Reply:
Sure, but I usually follow the Rule of three

https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)

Reply:
Haha, never heard of that, but I love it. I always followed the rule of two I spose.